packages/dashboard-core-plugins/src/panels/index.ts

```diff
@@ -1,4 +1,5 @@
 export { default as ChartPanel } from './ChartPanel';
+export * from './ChartPanel';
```

Comment:
Just a random note, I'm surprised this works correctly/exports the correct `ChartPanel`.
In `ChartPanel`, we've got `export class ChartPanel`, but then the export at the end is a default export that is a redux connected wrapped component. And for some reason it looks like that takes precedent? Not sure why that works but it seems to.

If we wanted to be really explicit, we'd probably export a named version of the connected component (e.g.
```
export const ConnectedChartPanel = connect(...)(ChartPanel);

export default ConnectedChartPanel;
```
), but not required for this change.